bucket/charm-gum.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "0.8.0",
+    "homepage": "https://charm.sh/",
+    "description": "A tool for glamorous shell scripts 🎀.",
+    "license": "MIT",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/charmbracelet/gum/releases/download/v0.8.0/gum_0.8.0_Windows_i386.zip",
+            "hash": "8da08c2125b35652f1204c43cee5713fad231deff1283664553b96d329378a18"
+        },
+        "64bit": {
+            "url": "https://github.com/charmbracelet/gum/releases/download/v0.8.0/gum_0.8.0_Windows_x86_64.zip",
+            "hash": "7c7f124975bf127f28006d1640371cb2d570240f76e82beeb536fb4b86cb8f0c"
+        }
+    },
+    "bin": [
+        "gum.exe",
+        [
+            "gum.exe",
+            "charm-gum"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/charmbracelet/gum"
+    },
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://github.com/charmbracelet/gum/releases/download/v$version/gum_$version_Windows_i386.zip"
+            },
+            "64bit": {
+                "url": "https://github.com/charmbracelet/gum/releases/download/v$version/gum_$version_Windows_x86_64.zip"
+            }
+        },
+        "hash": {
+            "url": "$baseurl/checksums.txt",
+            "regex": "($sha256)\\s+gum_[\\d.]+_Windows_*"

```

Comment:
Looks like it may cause hash update issue since there are `.sbom` files for each binary in release asset and corresponding entries in checksums. The current regex may pick up hash for `$basename.sbom` instead of `$basename` itself, if the entry with `.sbom` appears in checksums first (in further releases).

Please look into #3182 e.g.

Reply:
Ok, what do you suggest I should do to solve it?

Reply:
Expression used for goreleaser manifest [here](https://github.com/ScoopInstaller/Main/pull/3182/commits/e0a777514c98f8c77f2115abda030efecddef9c0) - `"regex": "$sha256  $basename\\n"` - works fine and lets avoid `hash` section duplication for each architecture section.

But your solution in update below looks workable as well.